token-metadata/program/src/instruction.rs

```diff
@@ -0,0 +1,120 @@
+use {
+    borsh::{BorshDeserialize, BorshSerialize},
+    solana_program::{
+        instruction::{AccountMeta, Instruction},
+        pubkey::Pubkey,
+        sysvar,
+    },
+    std::str::FromStr,
+};
+
+#[repr(C)]
+#[derive(BorshSerialize, BorshDeserialize, PartialEq, Debug, Clone)]
+/// Args for Create call
+pub struct CreateMetadataAccountArgs {
+    /// The name of the asset
+    pub name: String,
+    /// The symbol for the asset, ie, AAPL or SHOES
+    pub symbol: String,
+    /// URI pointing to JSON representing the asset
+    pub uri: String,
+}
+
+#[repr(C)]
+#[derive(BorshSerialize, BorshDeserialize, PartialEq, Debug, Clone)]
+/// Args for init call
+pub struct InitMetadataAccountArgs {
```

Comment:
Why have a duplicate struct?  Generalize the `CreateMetadataAccountArgs` name perhaps

Reply:
actually is this struct even used?

Reply:
oh no, i combined this functionw ith the other...good catch. Forgot to remove it.